### DISCUSSION TITLE
How do you get the github pro pack for students?

### DISCUSSION THREAD

**Discussion by thedrinkingmaster** [community/community#68988] [GitHub Education]
### Hi everyone,

I am a student and i would like to have a pro pack for my account

**Comment by RanvilleCarlo** [comment]
yes you can upgrade your account through pro mode using a website called github.education there you are able to update your GitHub free acoi=unt to GitHub pro

**Comment by supuna97** [comment]
Join this. https://education.github.com/

**Comment by queenofcorgis** [comment]
GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. Please note any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
